pan-domain-auth-play/src/main/scala/com/gu/pandomainauth/service/OAuth.scala

```diff
@@ -96,8 +97,8 @@ class OAuth(config: OAuthSettings, system: String, redirectUrl: String)(implicit
                   ),
                   authenticatingSystem = system,
                   authenticatedIn = Set(system),
-                  jwt.claims.exp * 1000,
-                  false
+                  expires = Instant.ofEpochSecond(jwt.claims.exp), // https://stackoverflow.com/a/39926886/438886
+                  multiFactor = false
```

Comment:
As explained in this [Stackoverflow answer](https://stackoverflow.com/a/39926886/438886), RFC 7519 [states that](https://www.rfc-editor.org/rfc/rfc7519#section-4.1.4) `exp` is a `NumericDate`, and that `NumericDate` is the number of **seconds** (not milliseconds) since the epoch - which is why we were multiplying by `1000` to get to the number of milliseconds for `expires` in the old code.

Now that `expires` is a strongly-typed `Instant`, we can use the explicit `Instant.ofEpochSecond()` method to make clear what conversion we're using.